vaadin-button-flow-parent/vaadin-button-flow-integration-tests/src/main/java/com/vaadin/flow/component/button/tests/TooltipDefaultsPage.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2022 Vaadin Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.vaadin.flow.component.button.tests;
```

Comment:
Would it make sense to introduce separate IT package so that we could decouple ITs for `Tooltip` features from the button package? Not a blocker, can be done separately but let's at least create an issue for it.

Reply:
Yeah, that was considered but there's no IT module at all in the `vaadin-flow-components-shared-parent` at the moment. That's why I needed to use some other component for the ITs for now